tests/README.md

```diff
@@ -1,24 +1,110 @@
 # Integration tests
 
-Integration tests require a running Kubevirt cluster.  Once you have a running
-Kubevirt cluster, you can use the `-master` and the `-kubeconfig` flags to
-point the tests to the cluster.
+Kubevirt integration tests are a suite of end-to-end tests which run
+against an active Kubevirt cluster.
 
-## Run them on an arbitrary KubeVirt installation
+The tests may execute against various clusters, from integrated kubevirtci
+deployments and up to external kubernetes or OpenShift ones.
+
+## Kubevirtci
+
+[Kubevirtci](https://github.com/kubevirt/kubevirtci/)
+provides containerized Kubernetes clusters which can be used
+to test KubeVirt and similar projects.
 
+KubeVirt integration tests are being run against kubevirtci clusters on CI
+and is the simplest way to run the tests locally in a development environment.
+
+To run the test locally:
+- Configure the desired cluster setup (provider, number of nodes and nics).
+  Note: Some tests may run only on specific configuration and hardware.
```

Comment:
the notes appear on the same line. please separate. 

(if you want to see the output readme locally, you can use [grip](https://github.com/joeyespo/grip). alternatively, you can use the github gui)